src/smf/packet-smf.c

```diff
@@ -784,10 +784,26 @@ static guint32 test_smf(tvbuff_t *tvb, packet_info* pinfo, int offset)
     }
     // Check protocol
     guint8 secondByte = tvb_get_guint8(tvb, offset+1);
-    if ((secondByte & 0x3f) > SMF_PROTOCOL_MAX)
+    guint8 smfProtocol = secondByte & 0x3f;
+    if (smfProtocol > SMF_PROTOCOL_MAX)
     {
         return 1;
     }
+    // Detect obsolete protocols 
+    switch (smfProtocol) {
+    case SMF_CSMP:
+    case SMF_PUBMSG:
+    case SMF_XMLLINK:
+    case SMF_WSE:
+    case SMF_SUBCTRL:
+    case SMF_PUBCTRL:
+    case SMF_KEEPALIVE:
+        g_print("smf obsolete protocol detected in packet %d, protocol %s(%d)\n", pinfo->fd->num, 
+            try_val_to_str(smfProtocol, protocolnames), smfProtocol);
```

Comment:
Yes, it will spam the console and tell me which frame it happens on.
I could add an exception error instead (so that it appears on the wireshark frame), but I am lazy.
Would you like me to add that instead?

Reply:
As  I understand it, we're just skipping over bytes that are not the start of a SMF packet while searching for the first SMF packet.  Is this correct?  In this case it's not an error or exception at all,  it's just normal processing when we're starting a pcap in the middle of a stream and doesn't warrant an error or exception at all.  Maybe a 'finest' or 'debug' log if such a think exists but even that would slow down our search.

Reply:
You are right. I will remove it.